cmake/NodeJS.cmake

```diff
@@ -595,7 +595,7 @@ function(add_nodejs_module NAME)
         PUBLIC ${NODEJS_DEFINITIONS}
     )
     # This properly defines includes for the module
-    target_include_directories(${NAME} PUBLIC ${NODEJS_INCLUDE_DIRS})
+    target_include_directories(${NAME} SYSTEM PUBLIC ${NODEJS_INCLUDE_DIRS})
```

Comment:
This is upstream code vendored from node-cmake (because they had an issue, should be fixed now with https://github.com/cjntaylor/node-cmake/pull/24) - the next update will remove your changes here again